tsdb/chunkenc/float_histogram.go

```diff
@@ -1088,6 +1089,20 @@ func (it *floatHistogramIterator) Next() ValueType {
 		} else {
 			it.nBuckets = nil
 		}
+		if len(it.pSpans) > 0 {
```

Comment:
Please do the same for integer histograms to keep them in sync.

Reply:
Done